python/jupytergis_lab/style/base.css

```diff
@@ -419,6 +419,9 @@ div.jGIS-toolbar-widget > div.jp-Toolbar-item:last-child {
 }
 
 /* Support output views, created with "Create new view for cell output" context menu */
+.jp-LinkedOutputView .jp-OutputArea-child:only-child {
+  height: 100%;
+}
```

Comment:
We could maybe add a note about https://github.com/jupyterlab/jupyterlab/pull/17487 to remove this when it will not be necessary anymore.

Reply:
Sounds good! We'll probably want to keep it around for a while for compatibility, though. When we remove it, this feature will stop working correctly with older versions of jupyterlab.

Reply:
66c3cd6